llarp/path/path.cpp

```diff
@@ -707,12 +707,8 @@ namespace llarp
     Path::HandleDHTMessage(const dht::IMessage& msg, AbstractRouter* r)
     {
       MarkActive(r->Now());
-      routing::DHTMessage reply;
-      if(!msg.HandleMessage(r->dht(), reply.M))
-        return false;
-      if(reply.M.size())
-        return SendRoutingMessage(reply, r);
-      return true;
+      std::vector< std::unique_ptr< dht::IMessage > > msgs;
+      return msg.HandleMessage(r->dht(), msgs);
```

Comment:
I'm assuming the fact that `msgs` is temporary implicitly drops the replies. This is fairly subtle, so a comment might help. Alternatively, an overloaded `HandleMessage` that doesn't offer any replies would be more explicit.

Reply:
i did not mean to include that change